views/index.ejs

```diff
@@ -1,82 +1,82 @@
 <!DOCTYPE html>
 <html>
 
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="description" content="Baat Cheet is an anonymous chat app">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title><%= title %></title>
-    <%- include('head_imports'); %>
-  </head>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="description" content="Baat Cheet is an anonymous chat app">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title><%= title %></title>
+  <%- include('head_imports'); %>
```

Comment:
Could you make all the chnages that are related to indentation and styling (in the css file too) as a separate commit, and the changes related to this specific issue a separate commit? This would help better demarcate changes between specific commits.